{
  "comments": [
    {
      "key": {
        "uuid": "dc51e7a9_c25726f4",
        "filename": "vp9/decoder/vp9_decodeframe.c",
        "patchSetId": 1
      },
      "lineNbr": 1360,
      "author": {
        "id": 1002431
      },
      "writtenOn": "2016-05-25T17:40:20Z",
      "side": 1,
      "message": "Seems weird that we don\u0027t return here, but I see the same happening below so I guess that\u0027s normal.\n\nDoes calling vpx_internal_error set a flag that results in the libvpx user getting an error value returned from codec_decode or get_frame?",
      "revId": "75b6cfe1c50e749a0edb5460a491ca5ac947aff5",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "dc51e7a9_425b1617",
        "filename": "vp9/decoder/vp9_decodeframe.c",
        "patchSetId": 1
      },
      "lineNbr": 1360,
      "author": {
        "id": 1000893
      },
      "writtenOn": "2016-05-25T17:43:58Z",
      "side": 1,
      "message": "Yes, it uses a long jump to get error info back out to application.",
      "parentUuid": "dc51e7a9_c25726f4",
      "revId": "75b6cfe1c50e749a0edb5460a491ca5ac947aff5",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": false
    }
  ]
}